examples/redis/custom.conf.d/cachedb.conf

```diff
@@ -0,0 +1,8 @@
+server:
+  module-config: "validator cachedb iterator"
+
+cachedb:
+  backend: "redis"
+  redis-server-host: redis
+  redis-server-port: 6379
+  redis-expire-records: yes
```

Comment:
This file should have the `.example` extension otherwise it will be loaded by default.

Reply:
But this is in the examples directory, not in roofs_overlay - it should not be part of for the image, right?

Reply:
Of course, my mistake!